src/components/DarkLightButton/DarkLightButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+// import { ThemeContext } from '../../context/ThemeContext';
```

Comment:
As dicussed, let's remove all the comments once we finish with our parts :)

Reply:
I will use this one for sure, it's just not working now, so I commented that part :D